nightly-examples/README.md

```diff
@@ -1,13 +0,0 @@
-# nightly-only examples
```

Comment:
can we add any new examples moved from `nightly-examples` to the `examples` dir's README?

Reply:
Sure, will move them over.